<launch>

    <node pkg="giskardpy" type="giskard.py" name="giskard" output="screen">
        <param name="config" value="TiagoMujoco"/>
    </node>

    <node pkg="giskardpy" type="interactive_marker.py" name="giskard_interactive_marker" output="screen">
        <rosparam param="enable_self_collision">False</rosparam>
        <rosparam param="interactive_marker_chains">
            - [base_footprint, gripper_left_grasping_frame]
            - [base_footprint, gripper_right_grasping_frame]
            - [base_footprint, xtion_link]
        </rosparam>
    </node>

    <node pkg="giskardpy" type="move_base_simple_goal_diff_drive.py" name="my_move_base" output="screen" />

    <node pkg="iai_tiago_tools" type="giskard_e_stop.py" name="giskard_e_stop" output="screen" />

</launch>
